docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1341 @@
+    "\\end{matrix} \\right]\n",
+    "$$\n",
+    "\n",
+    "With Floquet calibration, every angle but $\\chi$ can be calibrated. In experiments, we have found these angles change when gates are run in parallel. Because of this, we perform FC on entire moments of two-qubits gates and return different calibrated angles for each. \n",
+    "\n",
+    "After calibrating, one needs to adjust for these angles in the circuit to be implemented. The simplest correction is for single-qubit $Z$ rotations and works by adding $R_z$ gates before and after the two-qubit gates in question. For many circuits, even this simplest correction can lead to a significant improvement in results. We provide methods for doing this in this notebook and analyze results for an example circuit.\n",
+    "\n",
+    "We do not attempt to correct the misaligned iSWAP rotation or the additional two-qubit phase in this notebook. This is a non-trivial task and we do currently have simple tools to achieve this. It is up to the user to correct for these as best as possible."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "-jOmHd7lLyDn"
+   },
+   "source": [
+    "*Note*: The Floquet calibration API and this documentation is ongoing work. The amount by which errors are reduced may vary from run to run and from circuit to circuit."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "pigDhoDrO7vr"
+   },
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "AXHWCnLzufVf"
+   },
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import cirq\n",
+    "except ImportError:\n",
+    "    print(\"installing cirq...\")\n",
+    "    !pip install git+https://github.com/quantumlib/Cirq\n",
+    "    print(\"installed cirq.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "6zlksjAlum6f"
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"Imports.\"\"\"\n",
+    "from typing import Iterable, List, Optional, Sequence\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "import cirq\n",
+    "import cirq.google as cg  # Contains the Floquet calibration tools."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "w_x706KvjDLC"
+   },
+   "source": [
+    "*Note*: In order to run on Google's Quantum Computing Service, an environment variable `GOOGLE_CLOUD_PROJECT` must be present and set to a valid Google Cloud Platform project identifier. If this is not satisfied, we default to an engine simulator."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "57Uo5dfDRgNl"
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "if \"GOOGLE_CLOUD_PROJECT\" in os.environ:\n",
```

Comment:
please follow the authentication cell from docs/tutorials/google/start.ipynb - that ensures a portable pattern that works from colab and jupyter notebooks